packages/btc/src/transaction/build.ts

```diff
@@ -362,6 +375,13 @@ export class TxBuilder {
     }
 
     if (amount < this.minUtxoSatoshi) {
+      // If the change is not enough to create a non-dust output, try collect more.
+      // - injectCollected=true, expect to put all (collected + amount) of satoshi as change
+      // - deductFromOutputs=false, do not collect satoshi from the outputs
+      // An example:
+      // 1. Expected to return change of 500 satoshi, amount=500
+      // 2. Collected 2000 satoshi from the "fromAddress", collected=2000
+      // 3. Create a change output and return (collected + amount), output.value=2000+500=2500
       const { collected } = await this.injectSatoshi({
```

Comment:
```suggestion
      // If the change is not enough to create a non-dust output, try collect more.
      // - injectCollected=true, expect to put all (collected + amount) of satoshi as change
      // - deductFromOutputs=false, do not collect satoshi from the outputs
      // An example:
      // 1. Expected to return change of 500 satoshi, amount=500
      // 2. Collected 2000 satoshi from the "fromAddress", collected=2000
      // 3. Create a change output and return (collected + amount), output.value=2000+500=2500
      //
      // TODO(Low-Priority):
      // If the cost of creating a non-dust output is greater than the change amount, it may be
      // reasonable to just pay the change amount as an additional fee to the miners.
      const { collected } = await this.injectSatoshi({
```

Reply:
Raised an issue for the suggestion instead: https://github.com/ckb-cell/rgbpp-sdk/issues/183